devices.js

```diff
@@ -14855,6 +14855,29 @@ const devices = [
             await reporting.currentPositionLiftPercentage(endpoint);
         },
     },
+    {
+        zigbeeModel: ['LK Switch'],
+        model: '545D6514',
+        vendor: 'Schneider Electric',
+        description: 'LK FUGA Wiser Wireless Double Relay',
+        meta: {multiEndpoint: true, configureKey: 1},
+        fromZigbee: [fz.on_off, fz.wiser_fuga_relay_command],
```

Comment:
Instead of `fz.wiser_fuga_relay_command` please re-use `fz.command_on, fz.command_off`.

Reply:
Hi @Koenkk, thanks for your review!

The reason I didn't use the existing `fz.command_on` and `fz.command_off` is that each button will only generate one of the types `commandOn` or `commandOff` on a separate endpoint. For example, pushing each button once generates:

```
Zigbee2MQTT:debug 2021-03-12 20:52:34: Received Zigbee message from '0x60a423fffe50109c', type 'commandOn', cluster 'genOnOff', data '{}' from endpoint 21 with groupID 0
Zigbee2MQTT:debug 2021-03-12 20:52:36: Received Zigbee message from '0x60a423fffe50109c', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 22 with groupID 0
Zigbee2MQTT:debug 2021-03-12 20:52:38: Received Zigbee message from '0x60a423fffe50109c', type 'commandOn', cluster 'genOnOff', data '{}' from endpoint 23 with groupID 0
Zigbee2MQTT:debug 2021-03-12 20:52:41: Received Zigbee message from '0x60a423fffe50109c', type 'commandOff', cluster 'genOnOff', data '{}' from endpoint 24 with groupID 0
```


Using `fz.command_on` and `fz.command_off` the resulting action will be `on_s1`, `off_s2`, `on_s3`, `off_s4` (if I understand the converters right). I thought that could be confusing, since it really is 4 separate push buttons. If you think this is fine, just let me know and I will correct the PR to use `fz.command_on` and `fz.command_off`.

Thanks! :)

Reply:
I see, but with the current code you send a `single` for both `on` and `off`, doesn't it make sense to be able to use both events?

Reply:
Yes, ideally, the "problem" is e.g. `button 1` (i.e. top/left) always sends `on` and never `off`, while e.g. `button 2` (top/right) always sends `off` but never `on`. 

Anyhow, I just corrected the PR to re-use `fz.command_on` and `fz.command_off`, so there is no need for additions in `fromZigbee.js`